qutebrowser/browser/shared.py

```diff
@@ -273,10 +273,17 @@ def get_tab(win_id, target):
     return tabbed_browser.tabopen(url=None, background=bg_tab)
 
 
-def get_user_stylesheet():
-    """Get the combined user-stylesheet."""
+def get_user_stylesheet(url=None):
+    """Get the combined user-stylesheet.
+
+    If `url` is given and there's no overridden stylesheet, return
+    `configutils.UNSET`.
+    """
     css = ''
-    stylesheets = config.val.content.user_stylesheets
+    stylesheets = config.instance.get('content.user_stylesheets', url,
+                                      fallback=url is None)
+    if stylesheets is configutils.UNSET:
+        return stylesheets
```

Comment:
It looks like this `return` breaks `config.val.scrolling.bar` behavior below (line 292).